packages/babel-plugin-transform-node-env-inline/__tests__/node-env-inline-test.js

```diff
@@ -14,7 +14,7 @@ describe("node-env-inline-plugin", () => {
     const prev = process.env.NODE_ENV;
     process.env.NODE_ENV = "development";
 
-    const source = "process.env.NODE_ENV === \"development\";";
+    const source = 'process.env.NODE_ENV === "development";';
```

Comment:
JSON is double quotes. So I think prettier chooses single outside double instead of the other way around.